### ISSUE TITLE
Rooster Editor Alt Text Feature

### ISSUE THREAD

**Issue by v-adbarve** [microsoft/roosterjs#2719]
**Bug Report: Rooster Editor Alt Text Feature**

**Product**: Rooster Editor (React Web Application Integration) Version: 8.61.0 Issue: Alt text added to images is not visible for editing in the hosted Azure environment.

**Description**: After integrating the Rooster editor into our React Web Application, we observed that the alt text feature for images is not functioning as expected. While users can add alt text to images, which is then reflected in the HTML markup, the text is not available for editing when the image is right-clicked again. This issue was not present in the local development environment after upgrading to version 8.61.0, but persists in the Azure hosted environment.

**Steps to Reproduce:**

Upload an image in the Rooster editor.
Right-click on the image and add alt text.
Confirm that the alt text is present in the HTML markup.
Right-click on the image again to edit the alt text.
Observe that the previously added alt text is not visible in the dialog.
Expected Behavior: The alt text should be visible and editable when the image is right-clicked after the initial addition.

**Actual Behavior**: The alt text is not visible for editing upon subsequent accesses of the alt text dialog.

**Impact**: This issue impacts the accessibility of images within the application, as users are unable to edit alt text to ensure it accurately describes the images.

**Additional Information:** The issue was resolved in the local development environment upon upgrading to version 8.61.0 of Rooster editor. However, the same upgrade did not resolve the issue in the Azure hosted environment.

<img width="631" alt="AltTextIssue" src="https://github.com/microsoft/roosterjs/assets/167873853/88350cf9-4245-4779-b543-df89728a0831">

**Comment by JiuqingSong** [comment]
If you can provide a debug environment we can debug. But, please notice:
1. We don't do more bug fix for version 8.x, please use 9.x instead.
2. It seems you are using roosterjs-react. This is a package for demo and prototype only. We don't do bug fixes since it is not designed for production code. We recommend to build your own UI wrapper instead.
3. With your own UI wrapper you can get the alt text directly, no need to rely on roosterjs.

**Comment by v-adbarve** [comment]
Dear @JiuqingSong ,

Thank you for your valuable suggestions. I will proceed with implementing version 9 or above as recommended. In addition, could you please provide an example of the user interface wrapper that you mentioned? A formal example would greatly assist me in understanding and applying your suggestion more effectively.

Thank you for your assistance.

Best regards, Aditya Barve

**Comment by JiuqingSong** [comment]
For example:

```ts
import { EditorAdapter, EditorAdapterOptions } from 'roosterjs-editor-adapter';
import { EditorPlugin as NewEditorPlugin } from 'roosterjs-content-model-types';
import { EditorPlugin as LegacyEditorPlugin } from 'roosterjs-editor-types';


const oldPlugins: LegacyEditorPlugin[] = []
const newPlugins: NewEditorPlugin[] = [];
const options: EditorAdapterOptions = {
    'plugins': newPlugins,
    legacyPlugins: oldPlugins,
    ...
};

const div: HTMLDivElement = ...
const editor = new EditorAdapter(div, options);
```

Here you can declare both old style plugins (v8.x) and new style plugins (v9.x), and then use the wrapper class `EditorAdapter` which implements both v8 and v9 editor interfaces.
